tests/run/literals.scala

```diff
@@ -1,32 +1,38 @@
+// scalac: -deprecation
+//
 //############################################################################
 // Literals
 //############################################################################
 
-import scala.util.{Failure, Success, Try}
+//############################################################################
 
 object Test {
 
-  /* I add a couple of Unicode identifier tests here "temporarily" */
-
-  def \u03b1\u03c1\u03b5\u03c4\u03b7 = "alpha rho epsilon tau eta"
-
-  case class GGG(i: Int) {
-    def \u03b1\u03b1(that: GGG) = i + that.i
+  def check_success[A](name: String, closure: => A, expected: A): Unit = {
+    val res: Option[String] =
+      try {
+        val actual: A = closure
+        if (actual == expected) None  //print(" was successful")
+        else Some(s" failed: expected $expected, found $actual")
+      } catch {
+        case exception: Throwable => Some(s" raised exception $exception")
+      }
+    for (e <- res) println(s"test $name $e")
```

Comment:
I was following up that the unicodes on line 11 were edited out, when I noticed that this reverts my last change.

Reply:
The test has to assert because there is no longer a check file -- vulpix only checks the output if there is a check file.

Reply:
I'll see if I can restore those changes. Does vulpix detect a test failing an assertion? I guess I'll jiggle with it a bit and let you know.

As for the identifier on line 11, the variant that is still supported is in tests/run/unicodeEscapes.scala line 13-14, which makes good on the promise of temporarily.

Reply:
NBD, I PR'd it, thanks. I was about to futz with more parsing. It's amazing how quickly I forgot like how do I even run a test?